components/ModalDelete/ModalDelete.styled.js

```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components";
+
+export const ModalContainer = styled.div`
```

Comment:
I'm not sure but since the modal is a floating element over other content, you should use a z-index to make it appear above all other page content.